autoreject/autoreject.py

```diff
@@ -162,6 +181,8 @@ def get_rejection_threshold(epochs, decim=1, random_state=None, verbose=True):
         The decimation factor.
     random_state : int seed, RandomState instance, or None (default)
         The seed of the pseudo random number generator to use.
+    cv : int
+        The numbef of folds used. Defaults to 5.
```

Comment:
```suggestion
        The number of folds used. Defaults to 5.
```

Reply:
I don't have permission to commit from the github interface. I'll just commit it locally and mark you as coauthor